src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/emptylineseparator/InputEmptyLineSeparatorBlockCommentSeparatedFromPackage.java

```diff
@@ -0,0 +1,9 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.emptylineseparator;
+
+/*
+* My important message
```

Comment:
wan we use our approach in Inputs to use comments as place for violation ?  `//warn` `/* warn */`
with definition of config above class.

in all other inputs too.

Reply:
done